src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/EnvelopeSummaryResponse.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.blobrouter.model.out;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public class EnvelopeSummaryResponse {
+    public final String container;
+    public final String fileName;
+    public final LocalDate dateReceived;
+    public final LocalTime timeReceived;
```

Comment:
do we need to split it in the API response model?

Reply:
This is how it is done in processor. It makes sense in my opinion

Reply:
ok